src/sidebar/store/modules/defaults.js

```diff
@@ -0,0 +1,59 @@
+import * as util from '../util';
+
+/**
+ * A store module pertaining to client-side defaults for (browser) users.
+ */
+
+function init() {
+  /**
+   * Note that the persisted presence of any of these defaults cannot be
+   * guaranteed, so consumers of said defaults should be prepared to handle
+   * missing (i.e. `null`) values. As `null` is a sentinal value indicating
+   * "not set/unavailable", a `null` value for a default is otherwise invalid.
+   */
+  return {
+    annotationPrivacy: null,
+  };
+}
+
+const update = {
+  SET_DEFAULT: function(state, action) {
+    return { [action.defaultKey]: action.value };
+  },
+};
+
+const actions = util.actionTypes(update);
+
+function setDefault(defaultKey, value) {
+  return { type: actions.SET_DEFAULT, defaultKey: defaultKey, value: value };
+}
+
+/** Selectors */
+
+/**
+ * Retrieve the state's current value for `defaultKey`.
+ *
+ * @return {*} - The current value for `defaultKey` or `undefined` if it is not
+ *               present
+ */
+function getDefault(state, defaultKey) {
```

Comment:
At present, this module doesn't give a hoot about non-existent properties; it will happily return `undefined`. But at least that's differentiated from `null`, which has meaning here.

Reply:
Sure; I had considered doing that in my first go around. Given that I have an immediate follow-on PR to this one, I may make that change in that body of changes as it would avoid introducing a change/potential breakage into this, approved, PR.